Std/Data/List/Basic.lean

```diff
@@ -823,6 +823,35 @@ inductive Forall₂ (R : α → β → Prop) : List α → List β → Prop
 
 attribute [simp] Forall₂.nil
 
+@[simp] theorem forall₂_cons {R : α → β → Prop} {a b l₁ l₂} :
+    Forall₂ R (a :: l₁) (b :: l₂) ↔ R a b ∧ Forall₂ R l₁ l₂ :=
+  ⟨fun | .cons h tail => ⟨h, tail⟩, fun ⟨head, tail⟩ => .cons head tail⟩
+
+/--
+Check if all elements in the first List are `R` related to the element at the same position in the
+second List.
+-/
+def all₂ (R : α → β → Prop) [∀ a b, Decidable (R a b)] : List α → List β → Bool
```

Comment:
I believe the suggestion was this:
```lean
def all₂ (r : α → β → Bool) : List α → List β → Bool
```
Please avoid amending reviewed commits, that makes it hard to track how review comments were addressed.

Reply:
Ah ok, I wasn't aware. And I guess I kinda misread the suggestion 😅